### PR TITLE
Adding greating

### PR DESCRIPTION
**Pull request by marinaiftimescu**

### CODE REVIEW

File: Text-1.txt [hunk 1]
```diff
@@ -2,5 +2,7 @@ Hello :)
 
 
 
+
```
Comment (danciujoelle) [inline Text-1.txt]: this empty line is not supposed to be here

File: Text-1.txt [hunk 2]
```diff
@@ -2,5 +2,7 @@ Hello :)
...
 Devs will say hi here:
 * Gabi
+* Marina
```
Comment (danciujoelle) [inline Text-1.txt:7]: good code styling

**Review by danciujoelle — CHANGES_REQUESTED** [review]

**Review by cojocariubogdan — APPROVED** [review]

**Review by fcordos — APPROVED** [review]

**Review by gabidj — APPROVED** [review]

File: Text-1.txt (hunk 2 above)
Comment (axelspade) [reply Text-1.txt:7]: agreed with Joelle

**Review by axelspade — APPROVED** [review]

File: Text-1.txt (hunk 1 above)
Comment (danciujoelle) [reply Text-1.txt]: i will remove the line

File: Text-1.txt (hunk 2 above)
Comment (danciujoelle) [reply Text-1.txt:7]: thank you

File: Text-1.txt (hunk 1 above)
Comment (marinaiftimescu) [reply Text-1.txt]: oki

**Review by danciujoelle — APPROVED** [review]

**Pull request merged**